crates/rattler_conda_types/src/version/bump.rs

```diff
@@ -0,0 +1,26 @@
+use std::fmt;
+
+/// VersionBumpType is used to specify the type of bump to perform on a version.
+pub enum VersionBumpType {
+    /// Bump the major version number.
+    Major,
+    /// Bump the minor version number.
+    Minor,
+    /// Bump the patch version number.
+    Patch,
+    /// Bump the last  version number.
+    Last,
+}
+
+/// VersionBumpError is used to specify the type of error that occurred when bumping a version.
+#[derive(Debug, Clone)]
+pub struct VersionBumpError {
```

Comment:
Please use thiserror for error types.

Reply:
I refactored `VersionBumpError`. let me know if it's good enough.